docroot/modules/custom/va_gov_backend/src/Plugin/Validation/Constraint/OwnerUndefined.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Drupal\va_gov_backend\Plugin\Validation\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Checks that the owner isn't set to none.
+ *
+ * @Constraint(
+ *   id = "OwnerUndefined",
```

Comment:
```suggestion
 *   id = "FieldAdministrationUndefined",
```
field name is a more enduring source of truth.